apps/pie-microsite/src/_11ty/shortcodes/tokensTable/handleTokenData.js

```diff
@@ -1,6 +1,21 @@
 const pieTokensMetadata = require('../../../tokensMetadata.json');
-const pieTokenCategories = require('../../../tokenCategories.json');
 const { objectHelpers } = require('../../../utilities/helpers');
+const tokenTypes = require('../../../_data/tokenTypes');
+
+/**
+ * Gets the correct column size for each example based on token type.
+ * @param {string} tokenType - - the type of token i.e. color, spacing, radius
+ * @returns {string} - css variable containing correct column size
+ */
+const getExampleColumnSize = tokenType => {
+    const tokenColumnHandler = {
```

Comment:
I know its already been merged but I don't think handler is the most accurate name here. I'd have probably opted for something like `tokenExampleWidths`